examples/tutorials/advance_model_config/01_microbatch.ipynb

```diff
@@ -0,0 +1,250 @@
+{
```

Comment:
<p>It's highly confusing. Don't split relevant data across multiple cells (BATCH_SIZE, model_config, microbatch size, model config update). Simplify things!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>I changed the structure of the microbatch tutorial. Now it is simpliest way&nbsp;to demonstrate <code>microbatch</code> influence.</p>